src/Microsoft.AspNetCore.Mvc.Core/Internal/MvcCoreLoggerExtensions.cs

```diff
@@ -368,6 +387,102 @@ static MvcCoreLoggerExtensions()
                 LogLevel.Debug,
                 5,
                 "Current action does not explicitly specify any content types for the response.");
+
+            _registeredModelBinderProviders = LoggerMessage.Define<IModelBinderProvider[]>(
+                LogLevel.Debug,
+                6,
+                "Registered model binder providers, in the following order: {ModelBinderProviders}");
+
+            _tryingToBindProperty = LoggerMessage.Define<string, Type, Type, string>(
+               LogLevel.Debug,
+               7,
+               "Trying to bind property '{PropertyName}'[{ModelType}] on type '{PropertyContainerType}' using the name '{ModelName}' in request data ...");
```

Comment:
"on type" -> "of type" or "in type" throughout PR. Be careful to use "of type" when referring to the property or parameter's type and "in type" (or perhaps "in containing type") when referring to a member of a complex type.